frontend/src/metabase/components/Modal/utils.tsx

```diff
@@ -27,6 +27,6 @@ export function getModalContent(props: any) {
   ) {
     return <ModalContent {..._.omit(props, "className", "style")} />;
   } else {
-    return React.Children.only(props.children);
+    return props.children;
```

Comment:
For whatever reason, the FocusTrap doesn't seem to work specifically for `ObjectDetail` if I use `React.Children.only` versus just returning `props.children`. I logged both the objects and compared, and they seem like they are the same. It's super weird and I can't replicate it with the other modals that the FocusTrap affects.

Just for reference:
According to the React docs, `Children.only`: 
```
Verifies that children has only one child (a React element) and returns it. Otherwise this method throws an error.
```

We already check if the React.Children.count > 1, so I'm not sure that this is needed